etc/adminhtml/system.xml

```diff
@@ -21,178 +21,245 @@
                         <label>Publishable Key - Multi Step</label>
+                        <tooltip>Used by Bolt JavaScript to display the Multistep Experience</tooltip>
+                        <config_path>payment/boltpay/publishable_key_checkout</config_path>
                         <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
                     </field>
                     <field id="publishable_key_payment" translate="label" type="obscure" sortOrder="60" showInDefault="1" showInWebsite="1" showInStore="0">
                         <label>Publishable Key - Payment Only</label>
+                        <tooltip>Used by Bolt JavaScript to display the Payment Only Experience</tooltip>
+                        <config_path>payment/boltpay/publishable_key_payment</config_path>
                         <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
                     </field>
                     <field id="publishable_key_back_office" translate="label" type="obscure" sortOrder="65" showInDefault="1" showInWebsite="1" showInStore="0">
                         <label>Publishable Key - Back Office</label>
+                        <tooltip>Used by Bolt JavaScript to display the Back Office Experience</tooltip>
+                        <config_path>payment/boltpay/publishable_key_back_office</config_path>
                         <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
                     </field>
-                    <field id="sandbox_mode" translate="label" type="select" sortOrder="70" showInDefault="1" showInWebsite="1" showInStore="0">
-                        <label>Sandbox Mode</label>
+                </group>
+                <group id="where" translate="label" type="text" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Where to add Bolt?</label>
+                    <attribute type="expanded">1</attribute>
+                    <field id="minicart_support" translate="label" type="select" sortOrder="220" showInDefault="1" showInWebsite="1" showInStore="0">
+                        <label>MiniCart</label>
+                        <config_path>payment/boltpay/minicart_support</config_path>
                         <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <comment>Display Bolt Checkout in the Mini Cart</comment>
                     </field>
-                    <field id="is_pre_auth" translate="label" type="select" sortOrder="85" showInDefault="1" showInWebsite="1" showInStore="0">
-                        <label>Pre-Auth order creation</label>
+                    <field id="enable_payment_only_checkout" translate="label" type="select" sortOrder="250" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Native Checkout Page</label>
```

Comment:
Just to double-check, does this wording make sense @danac-gs? It was "Enable Payment Only Checkout" before

Reply:
@vitaliyreznikov It was added to explicitly disable Bolt on some 3rd party _one-step / one-page checkouts_. We do not officially support native checkout, but there is code in plugin that was handling that (quote a while ago) and is abandoned / deprecated now. However, if I recall correctly, @johnmccombs1 worked on integrating it for a merchant with a specific need. Not sure how that went through. Anyway, the wording seems fine.